tests/integration/test_examples.py

```diff
@@ -237,3 +239,50 @@ def test_whylogs_profiling(examples_dir: Path):
     assert isinstance(output_obj, DatasetProfile)
     output_obj = run.get_step("test_data_profiler").output.read()
     assert isinstance(output_obj, DatasetProfile)
+
+
+def test_run_custom_materializer(examples_dir: Path):
+    """Testing the functionality of the custom materializer example.
+
+    Args:
+        Temporary folder containing all examples including the run_examples
```

Comment:
Here and below should specify for which argument the description is, `examples_dir: Temporary ...`

Reply:
Good catch - this was throughout the file even on the other tests! #copypasta!